test/test.go

```diff
@@ -45,7 +47,8 @@ type Helper struct {
 
 // NewHelper initializes a new helper for testing.
 func NewHelper(t *testing.T) *Helper {
-	return &Helper{t: t}
+	wd, _ := os.Getwd()
```

Comment:
Also, just a note - `go test` will always set the cwd of the test process to the directory containing the test file that's running.